components/ThemeSwitch.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { useState, useEffect } from "react";
+import { useTheme } from "next-themes";
+
+export const ThemeSwitch = () => {
+  const [mounted, setMounted] = useState(false);
+  const { theme, setTheme } = useTheme();
+
+  // useEffect only runs on the client, so now we can safely show the UI
```

Comment:
This message is not that clear, can we improve it?

Suggestion:
```suggestion
  // useEffect only runs on the client, so  it's safe to show the UI
```

Reply:
Updated!